test/jdk/java/lang/invoke/ClassValueTest.java

```diff
@@ -479,4 +477,67 @@ protected Integer computeValue(Class<?> type) {
         awaitThreads(t);
         assertEquals(42, clv.get(int.class), "slow computation reinstalled value");
     }
+
+    // ClassValue cache invalidated and not reinstated when another
+    // unrelated entry is removed
+    @Test
+    public void cacheRefreshTest() throws Throwable {
```

Comment:
All other test methods in this class use test* naming. Let's follow it for consistency